### ISSUE TITLE
Early Stop

### ISSUE THREAD

**Issue by stanley-tsgs** [Nixtla/neuralforecast#1105]
### Description

I couldn't find the related documentation for implementing "early_stop".

**Comment by marcopeix** [comment]
All models have the parameter `early_stop_patience_steps`. By default it's -1, meaning it will not early stop. If you set it to 3, then training stop if the validation loss doesn't improve three times in a row. 

This parameter is documented for all models. [Here](https://nixtlaverse.nixtla.io/neuralforecast/models.autoformer.html#2-autoformer)'s where you can find for Autoformer, for example.

**Comment by stanley-tsgs** [comment]
Hi Marcopeix, thx a lot for the reply. I will try and feedback if there is any further issue regarding this topic.

**Comment by stanley-tsgs** [comment]
@marcopeix Hi Marco, I got below error: ERROR - An error occurred: Early stopping conditioned on metric `ptl/val_loss` which is not available. Pass in or modify your `EarlyStopping` callback to use any of the following: `train_loss`, `train_loss_step`, `train_loss_epoch`.
